From: Qingqing <wangyiqing@kylinos.cn>
Date: Tue, 21 Jul 2026 09:45:17 +0000
Subject: Translated using Weblate (Lao)

Currently translated at 100.0% (91 of 91 strings)

Translation: openkylin-huanghe/peony-qt-desktop
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/peony-qt-desktop/lo/
---
 .../peony-qt-desktop/peony-qt-desktop_lo.ts        | 174 ++++++++++-----------
 1 file changed, 87 insertions(+), 87 deletions(-)

diff --git a/translations/peony-qt-desktop/peony-qt-desktop_lo.ts b/translations/peony-qt-desktop/peony-qt-desktop_lo.ts
index 254f400..d8bb9ea 100644
--- a/translations/peony-qt-desktop/peony-qt-desktop_lo.ts
+++ b/translations/peony-qt-desktop/peony-qt-desktop_lo.ts
@@ -7,7 +7,7 @@
         <location filename="../../peony-qt-desktop/desktop-window-manager.cpp" line="640"/>
         <location filename="../../peony-qt-desktop/desktop-window-manager.cpp" line="645"/>
         <source>New Partition</source>
-        <translation type="unfinished"></translation>
+        <translation>ສ້າງພາກໃໝ່</translation>
     </message>
 </context>
 <context>
@@ -15,7 +15,7 @@
     <message>
         <location filename="../../peony-qt-desktop/controls/file-action-controler.cpp" line="393"/>
         <source>New Folder</source>
-        <translation type="unfinished"></translation>
+        <translation>ສ້າງໂຟນເດີໃໝ່</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/controls/file-action-controler.cpp" line="501"/>
@@ -30,22 +30,22 @@
     <message>
         <location filename="../../peony-qt-desktop/controls/file-action-controler.cpp" line="546"/>
         <source>Open Link failed</source>
-        <translation type="unfinished"></translation>
+        <translation>ເປີດ shortcut ລົ້ມເຫຼວ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/controls/file-action-controler.cpp" line="547"/>
         <source>File not exist, do you want to delete the link file?</source>
-        <translation type="unfinished"></translation>
+        <translation>ໂຟນເປົ້າໝາຍບໍ່ມີຢູ່, ຕ້ອງການລຶບທາງລັດທີ່ບໍ່ຖືກຕ້ອງບໍ່?</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/controls/file-action-controler.cpp" line="563"/>
         <source>Open failed</source>
-        <translation type="unfinished"></translation>
+        <translation>ເປີດບໍ່ສຳເລັດ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/controls/file-action-controler.cpp" line="564"/>
         <source>Open directory failed, you have no permission!</source>
-        <translation type="unfinished"></translation>
+        <translation>ເປີດໂຟນເດີບໍ່ສຳເລັດ, ທ່ານບໍ່ມີສິດໃນ directory ນີ້!</translation>
     </message>
 </context>
 <context>
@@ -56,17 +56,17 @@
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="824"/>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="830"/>
         <source>Reverse Select</source>
-        <translation type="unfinished"></translation>
+        <translation>ກົດປິດ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="387"/>
         <source>New Folder</source>
-        <translation type="unfinished"></translation>
+        <translation>ສ້າງໂຟນເດີໃໝ່</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="847"/>
         <source>Select All</source>
-        <translation type="unfinished"></translation>
+        <translation>ເລືອກທັງໝົດ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="201"/>
@@ -74,39 +74,39 @@
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="316"/>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="327"/>
         <source>Open</source>
-        <translation type="unfinished"></translation>
+        <translation>ເປີດ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="226"/>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="312"/>
         <source>Open with...</source>
-        <translation type="unfinished"></translation>
+        <translation>ເປີດໂດຍໃຊ້</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="238"/>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="304"/>
         <source>More applications...</source>
-        <translation type="unfinished"></translation>
+        <translation>ແອັບຯອື່ນໆ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="287"/>
         <source>File Manager</source>
-        <translation type="unfinished"></translation>
+        <translation>ຕົວຈັດການຟາຍ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="333"/>
         <source>Open %1 selected files</source>
-        <translation type="unfinished"></translation>
+        <translation>ເປີດ %1 ໄຟລ໌ທີ່ເລືອກ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="361"/>
         <source>Store %1 selected files into a new partition</source>
-        <translation type="unfinished"></translation>
+        <translation>ຈັດເກັບ %1 ໄຟລ໌ໄປໃນພາກໃໝ່</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="375"/>
         <source>New</source>
-        <translation type="unfinished"></translation>
+        <translation>ສ້າງໃໝ່</translation>
     </message>
     <message>
         <source>Empty File</source>
@@ -115,173 +115,173 @@
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="382"/>
         <source>Folder</source>
-        <translation type="unfinished"></translation>
+        <translation>ໂຟນເດີ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="395"/>
         <source>Text Document</source>
-        <translation type="unfinished"></translation>
+        <translation>ເອກະສານຂໍ້ຄວາມ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="560"/>
         <source>Auto arrange</source>
-        <translation type="unfinished"></translation>
+        <translation>ຈັດເລີຍອັດຕະໂນມັດ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="569"/>
         <source>View Type</source>
-        <translation type="unfinished"></translation>
+        <translation>ປະເພດການເບິ່ງ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="575"/>
         <source>Small</source>
-        <translation type="unfinished"></translation>
+        <translation>ຮູບສັນຍາລັກນ້ອຍ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="578"/>
         <source>Normal</source>
-        <translation type="unfinished"></translation>
+        <translation>ໄອຄອນກາງ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="581"/>
         <source>Large</source>
-        <translation type="unfinished"></translation>
+        <translation>ໄອຄອນໃຫຍ່</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="584"/>
         <source>Huge</source>
-        <translation type="unfinished"></translation>
+        <translation>ຮູບສັນຍາລັກໃຫຍ່ຫຼາຍ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="612"/>
         <source>Sort By</source>
-        <translation type="unfinished"></translation>
+        <translation>ປະເພດການຈັດລຳດັບ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="618"/>
         <source>Time Modified</source>
-        <translation type="unfinished"></translation>
+        <translation>ເວລາແກ້ໄຂ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="652"/>
         <source>Newest to oldest</source>
-        <translation type="unfinished"></translation>
+        <translation>ຈາກໃໝ່ໄປເກົ່າ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="653"/>
         <source>Oldest to newest</source>
-        <translation type="unfinished"></translation>
+        <translation>ຈາກເກົ່າໄປໃໝ່</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="656"/>
         <source>Files from large to small</source>
-        <translation type="unfinished"></translation>
+        <translation>ຈາກໃຫຍ່ໄປຫານ້ອຍ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="657"/>
         <source>Files from small to large</source>
-        <translation type="unfinished"></translation>
+        <translation>ຈາກນ້ອຍໄປຫາໃຫຍ່</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="684"/>
         <source>Desktop Origanization</source>
-        <translation type="unfinished"></translation>
+        <translation>ຈັດລຽນ Desktop</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="697"/>
         <source>Create New Partiton</source>
-        <translation type="unfinished"></translation>
+        <translation>ສ້າງພາກໃໝ່</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="974"/>
         <source>Mount</source>
-        <translation type="unfinished"></translation>
+        <translation>ເຊື່ອມຕໍ່</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="1146"/>
         <source>Remove Desktop Widget</source>
-        <translation type="unfinished"></translation>
+        <translation>ຍ້າຍແວັດສະດອກເດັສທັອບອອກ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="1169"/>
         <source>Add Plugin</source>
-        <translation type="unfinished"></translation>
+        <translation>ເພີ່ມ plugin</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="617"/>
         <source>Name</source>
-        <translation type="unfinished"></translation>
+        <translation>ຊື່ຟາຍ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="619"/>
         <source>File Type</source>
-        <translation type="unfinished"></translation>
+        <translation>ປະເພດຂອງໄຟລ໌</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="620"/>
         <source>File Size</source>
-        <translation type="unfinished"></translation>
+        <translation>ຂະໜາດຟາຍ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="738"/>
         <source>Clean the trash</source>
-        <translation type="unfinished"></translation>
+        <translation>ລຶບຖັງຂີ້ເຫຍື້ອໃຫ້ວ່າງ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="769"/>
         <source>Copy</source>
-        <translation type="unfinished"></translation>
+        <translation>ສຳເນົາ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="808"/>
         <source>Delete to trash</source>
-        <translation type="unfinished"></translation>
+        <translation>ລຶບໄປຖັງຂີ້ເຫຍື້ອ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="763"/>
         <source>Cut</source>
-        <translation type="unfinished"></translation>
+        <translation>ຕັດ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="818"/>
         <source>Delete forever</source>
-        <translation type="unfinished"></translation>
+        <translation>ລຶບຖາວອນ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="777"/>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="783"/>
         <source>Rename</source>
-        <translation type="unfinished"></translation>
+        <translation>ປ່ຽນຊື່</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="153"/>
         <source>Open Desktop in Window</source>
-        <translation type="unfinished"></translation>
+        <translation>ເປີດ &quot;Desktop&quot; ໃນຫນ້າຕ່າງ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="837"/>
         <source>Paste</source>
-        <translation type="unfinished"></translation>
+        <translation>ວາງ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="703"/>
         <source>Refresh</source>
-        <translation type="unfinished"></translation>
+        <translation>ຣີເຟຣຊ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="862"/>
         <source>Properties</source>
-        <translation type="unfinished"></translation>
+        <translation>ຄຸນສົມບັດ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="649"/>
         <source>Ascending Order</source>
-        <translation type="unfinished"></translation>
+        <translation>ຈາກນ້ອຍໄປຫຼາຍ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="648"/>
         <source>Descending Order</source>
-        <translation type="unfinished"></translation>
+        <translation>ລຳດັບລົງ</translation>
     </message>
 </context>
 <context>
@@ -289,42 +289,42 @@
     <message>
         <location filename="../../peony-qt-desktop/peony-desktop-application.cpp" line="246"/>
         <source>peony-qt-desktop</source>
-        <translation type="unfinished"></translation>
+        <translation>ເດັສທັອບ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/peony-desktop-application.cpp" line="516"/>
         <source>Close the peony desktop window</source>
-        <translation type="unfinished"></translation>
+        <translation>ປິດໂປຣແກຣມ Desktop</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/peony-desktop-application.cpp" line="519"/>
         <source>Take over the dbus service.</source>
-        <translation type="unfinished"></translation>
+        <translation>ຮັບບໍລິການDBus.</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/peony-desktop-application.cpp" line="522"/>
         <source>Take over the desktop displaying</source>
-        <translation type="unfinished"></translation>
+        <translation>ຮັບເດັສທັອບ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/peony-desktop-application.cpp" line="525"/>
         <source>Setup backgrounds</source>
-        <translation type="unfinished"></translation>
+        <translation>ຕັ້ງຄ່າພື້ນຫຼັງ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/peony-desktop-application.cpp" line="528"/>
         <source>Clear standard icons</source>
-        <translation type="unfinished"></translation>
+        <translation>ລຶບໄອຄອນມາດຕະຖານ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/peony-desktop-application.cpp" line="531"/>
         <source>Layout item with top to bottom left to right</source>
-        <translation type="unfinished"></translation>
+        <translation>ຈັດຮຽງໄອຄອນຈາກຊ້າຍເທິງໄປຂວາລຸ່ມ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/peony-desktop-application.cpp" line="535"/>
         <source>Force update backgrounds</source>
-        <translation type="unfinished"></translation>
+        <translation>ບັງຄັບອັບເດດພື້ນຫຼັງ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/peony-desktop-application.cpp" line="539"/>
@@ -337,21 +337,21 @@
     <message>
         <location filename="../../peony-qt-desktop/qml-desktop/platformwindow.cpp" line="2869"/>
         <source>Application</source>
-        <translation type="unfinished"></translation>
+        <translation>ໂປຣແກຣມ</translation>
     </message>
     <message>
         <source>Desktop background</source>
-        <translation type="unfinished"></translation>
+        <translation>ພື້ນທີ່ຫວ່າງເປົ່າຂອງເດັສທັອບ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/qml-desktop/platformwindow.cpp" line="2872"/>
         <source>Folder</source>
-        <translation type="unfinished"></translation>
+        <translation>ໂຟນເດີ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/qml-desktop/platformwindow.cpp" line="2875"/>
         <source>Files</source>
-        <translation type="unfinished"></translation>
+        <translation>ຟາຍ</translation>
     </message>
 </context>
 <context>
@@ -359,12 +359,12 @@
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="987"/>
         <source>Background Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>ຕັ້ງຄ່າພື້ນຫຼັງ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-menu.cpp" line="993"/>
         <source>Display Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>ຕັ້ງຄ່າການສະແດງຜົນ</translation>
     </message>
 </context>
 <context>
@@ -372,38 +372,38 @@
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/widget-manager-dialog.cpp" line="802"/>
         <source>Desktop Widget Manager</source>
-        <translation type="unfinished"></translation>
+        <translation>ຕົວຈັດການແວັດສະດອກສຳລັບເດັສທັອບ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/widget-manager-dialog.cpp" line="916"/>
         <source>Search application cards</source>
-        <translation type="unfinished"></translation>
+        <translation>ຊອກຫາແຜ່ນ Application</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/widget-manager-dialog.cpp" line="992"/>
         <source>Click a card to add it to the desktop, or drag a card onto the desktop.</source>
-        <translation type="unfinished"></translation>
+        <translation>ຄລິກບັດເພື່ອເພີ່ມລົງໃນເດັສທັອບ, ຫຼື ລາກບັດໄປວາງໃນເດັສທັອບ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/widget-manager-dialog.cpp" line="995"/>
         <source>Done</source>
-        <translation type="unfinished"></translation>
+        <translation>ສຳເລັດ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/widget-manager-dialog.cpp" line="1044"/>
         <source>All</source>
-        <translation type="unfinished"></translation>
+        <translation>ທັງໝົດ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/widget-manager-dialog.cpp" line="1115"/>
         <source>No desktop widgets found.
 Install widget packages to get started.</source>
-        <translation type="unfinished"></translation>
+        <translation>ບໍ່ພົບ widget ຂອງ desktop. ກະລຸນາຕິດຕັ້ງ package ຂອງ widget ເພື່ອເລີ່ມໃຊ້.</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/widget-manager-dialog.cpp" line="1169"/>
         <source>All desktop widgets</source>
-        <translation type="unfinished"></translation>
+        <translation>ແວັດສະດອກເດັສທັອບທັງໝົດ</translation>
     </message>
 </context>
 <context>
@@ -411,82 +411,82 @@ Install widget packages to get started.</source>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/desktop-widgets-menu.cpp" line="12"/>
         <source>Rename Partition</source>
-        <translation type="unfinished"></translation>
+        <translation>ປ່ຽນຊື່ພາກ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/desktop-widgets-menu.cpp" line="16"/>
         <source>View Type</source>
-        <translation type="unfinished"></translation>
+        <translation>ວິທີການເບິ່ງ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/desktop-widgets-menu.cpp" line="17"/>
         <source>Icon View</source>
-        <translation type="unfinished"></translation>
+        <translation>ມຸມມອງຮູບສັນຍາລັກ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/desktop-widgets-menu.cpp" line="18"/>
         <source>List View</source>
-        <translation type="unfinished"></translation>
+        <translation>ມຸມມອງແບບລາຍຊື່</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/desktop-widgets-menu.cpp" line="26"/>
         <source>Sort Type</source>
-        <translation type="unfinished"></translation>
+        <translation>ວິທີຈັດລຳດັບ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/desktop-widgets-menu.cpp" line="28"/>
         <source>File Name</source>
-        <translation type="unfinished"></translation>
+        <translation>ຊື່ໄຟລ໌</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/desktop-widgets-menu.cpp" line="32"/>
         <source>Time Modified</source>
-        <translation type="unfinished"></translation>
+        <translation>ເວລາແກ້ໄຂ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/desktop-widgets-menu.cpp" line="36"/>
         <source>File Type</source>
-        <translation type="unfinished"></translation>
+        <translation>ປະເພດຟາຍ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/desktop-widgets-menu.cpp" line="40"/>
         <source>File Size</source>
-        <translation type="unfinished"></translation>
+        <translation>ຂະໜາດຂອງໄຟລ໌</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/desktop-widgets-menu.cpp" line="47"/>
         <source>Descending Order</source>
-        <translation type="unfinished"></translation>
+        <translation>ຈາກຫຼາຍໄປນ້ອຍ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/desktop-widgets-menu.cpp" line="48"/>
         <source>Ascending Order</source>
-        <translation type="unfinished"></translation>
+        <translation>ລຳດັບຂຶ້ນ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/desktop-widgets-menu.cpp" line="55"/>
         <source>Newest to oldest</source>
-        <translation type="unfinished"></translation>
+        <translation>ຈາກໃໝ່ໄປຫາເກົ່າ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/desktop-widgets-menu.cpp" line="56"/>
         <source>Oldest to newest</source>
-        <translation type="unfinished"></translation>
+        <translation>ຈາກເກົ່າໄປຫາໃໝ່</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/desktop-widgets-menu.cpp" line="59"/>
         <source>Files from large to small</source>
-        <translation type="unfinished"></translation>
+        <translation>ຈາກໃຫຍ່ໄປຫານ້ອຍ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/desktop-widgets-menu.cpp" line="60"/>
         <source>Files from small to large</source>
-        <translation type="unfinished"></translation>
+        <translation>ຈາກນ້ອຍໄປຫຼາຍ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/desktop-widgets-menu.cpp" line="80"/>
         <source>Disband Partition</source>
-        <translation type="unfinished"></translation>
+        <translation>ຍົກເລີກພາກ</translation>
     </message>
     <message>
         <location filename="../../peony-qt-desktop/desktop-widgets/base/desktop-widgets-menu.h" line="66"/>
